### PR TITLE
Move ex11 to SI

### PR DESCRIPTION
**Pull request by PaulWessel**
See #2471.

### CODE REVIEW

**Comment by PaulWessel** [comment]
Did I get them all?

File: doc/examples/ex11/ex11.bat [hunk 1]
```diff
@@ -20,61 +20,61 @@ gmt begin ex11
...
 	gmt plot -Wthinner,white,- @rays_11.txt
-	echo 128 128 135 12p 180\217 > tmp.txt
+	echo 128 128 135 12p 180@.	 > tmp.txt
 	echo 102  26  90 12p 0.4	>> tmp.txt
 	echo 204  26  90 12p 0.8	>> tmp.txt
 	echo 10  140   0 16p B		>> tmp.txt
 	gmt text tmp.txt --FONT=white -F+a+f
-	echo 0 0 0 128 | gmt plot -N -Sv0.15i+s+e -Gwhite -W2p,white
+	echo 0 0 0 128 | gmt plot -N -Sv0.4c+s+e -Gwhite -W2p,white
 	echo 0 0 128 0 | gmt plot -N -Sv0.15i+s+e -Gwhite -W2p,white
```
Comment (seisman) [inline doc/examples/ex11/ex11.bat]:
```suggestion
	echo 0 0 128 0 | gmt plot -N -Sv0.4c+s+e -Gwhite -W2p,white
```

File: doc/examples/ex11/ex11.sh [hunk 2]
```diff
@@ -20,67 +20,67 @@ gmt begin ex11
...
+	gmt grdimage x1.nc y1.nc c1.nc -JX6c/-6c -X1.25c -R0/255/0/255
 	gmt plot -Wthinner,white,- @rays_11.txt
 	gmt text --FONT=white -F+a+f <<- END
-	128 128 -45 12p 60\217
+	128 128 -45 12p 60@.
 	102  26 -90 12p 0.4
 	204  26 -90 12p 0.8
 	10  140 180 16p G
 	END
 	echo 0 0 0 128 | gmt plot -N -Sv0.15i+s+e -Gwhite -W2p,white
```
Comment (seisman) [inline doc/examples/ex11/ex11.sh]: One more at line 34.

Comment (PaulWessel) [reply doc/examples/ex11/ex11.sh]: Hm, I don't see it? I think I got that one.

**Pull request merged**